app/logical/source/extractor/lofter.rb

```diff
@@ -12,11 +12,21 @@ def image_urls
         if parsed_url.image_url?
           [parsed_url.full_image_url]
         else
-          images = page&.search(".imgclasstag img, .ct .txtcont img")
-          images.to_a.pluck("src").map { |url| Source::URL.parse(url).full_image_url }
+          images = [*images_from_photo_post, *images_from_text_post]
```

Comment:
Potential regression: <https://jiuhaotaiyangdeshexian.lofter.com/post/73f37cdf_2b86a4ae7> ([post #6128400](https://danbooru.donmai.us/posts/6128400)) doesn't have `photoPostView` or `textPostView` but does have `answerPostView` with an `images` field. This is the only post on Danbooru with this field. Another unhandled field is `videoPostView` ([examples](https://danbooru.donmai.us/posts?tags=source%253A*lofter.com*+video+status%253Aany+-bad_id)) but the current version of the extractor doesn't seem to support Lofter videos anyway.

Reply:
Fixed both answer and video posts